app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+name: Remote Execution Host Job
+description: Example payload for actions.remote_execution.run_host_job_succeeded
+snippet: false
+model: WebhookTemplate
+-%>
+#
+# Example webhook template, turn this into a XML, JSON, shell script etc.
+# For more information about available helpers visit /templates_doc
+#
+# Run job host name <%= @object.host_name %>
```

Comment:
Probably we should mention that `@object` here means `Action`, not `Task`.

Reply:
Done!